src/server/graphql/subscriptions/integrationLeft.js

```diff
@@ -0,0 +1,26 @@
+import {GraphQLID, GraphQLNonNull} from 'graphql';
+import makeSubscribeIter from 'server/graphql/makeSubscribeIter';
+import {requireSUOrTeamMember} from 'server/utils/authorization';
+import LeaveIntegrationPayload from 'server/graphql/types/LeaveIntegrationPayload';
+import IntegrationService from 'server/graphql/types/IntegrationService';
+
+export default {
+  type: new GraphQLNonNull(LeaveIntegrationPayload),
+  args: {
+    service: {
+      type: new GraphQLNonNull(IntegrationService)
+    },
+    teamId: {
+      type: new GraphQLNonNull(GraphQLID)
+    }
+  },
+  subscribe: (source, {service, teamId}, {authToken, socketId}) => {
+    // AUTH
+    requireSUOrTeamMember(authToken, teamId);
+
+    // RESOLUTION
+    const channelName = `integrationLeft.${teamId}.${service}`;
+    const filterFn = (value) => value.mutatorId !== socketId;
+    return makeSubscribeIter(channelName, filterFn);
```

Comment:
Ah, I see! It's places like _this_ is where those getPubSub published messages are consumed. That makes sense. Hmm... We're really starting to use less and less of socketcluster now, aren't we?

Any reason why you decided to use redis and not the sc exchange? If we favor using redis (which I am in support of) we should probably refactor any of the other places we use the exchange and then favor a single service.